nuxeo-admin-console-web/angular-app/src/app/features/sub-features/generic-multi-feature-layout/services/generic-multi-feature-utilities.service.ts

```diff
@@ -113,4 +117,42 @@ export class GenericMultiFeatureUtilitiesService {
     }
     return { requestUrl, requestParams };
   }
+
+  checkIfErrorHasResponse(err: unknown): boolean {
+    return (
+      typeof err === "object" &&
+      err !== null &&
+      "response" in err &&
+      typeof (err as { response: unknown }).response === "object" &&
+      (err as { response: { json: unknown } }).response !== null &&
+      "json" in (err as { response: { json: unknown } }).response &&
+      typeof (err as { response: { json: () => Promise<unknown> } }).response
+        .json === "function"
+    );
+  }
+  handleError(err: unknown): Promise<unknown> {
+    if (this.checkIfErrorHasResponse(err)) {
+      console.log("checkIfErrorHasResponse",err)
+      return (err as { response: { json: () => Promise<unknown> } }).response.json();
+    } else {
+      console.log("handleError",ERROR_MODAL_LABELS.UNEXPECTED_ERROR)
+
+      return Promise.reject(ERROR_MODAL_LABELS.UNEXPECTED_ERROR);
+    }
+  }
+
+  handleErrorJson(errorJson: unknown, actionType: 'document' | 'folder'): void {
```

Comment:
remove actionType, and use action -- FeatureActions.onDocumentActionFailure  or FeatureActions.onFolderActionFailure

Reply:
done